x-pack/plugins/apm/public/components/app/service_node_metrics/index.tsx

```diff
@@ -180,27 +168,26 @@ export function ServiceNodeMetrics({ match }: ServiceNodeMetricsProps) {
         </MetadataFlexGroup>
       )}
       <SearchBar />
-      <EuiPage>
```

Comment:
I've removed EuiPage everywhere. Everything is now wrapped in EuiPage automatically when using `ApmMainTemplate`